apps/app/src/components/GlobalStyle.js

```diff
@@ -19,6 +20,7 @@ export const GlobalStyle = createGlobalStyle`
     height: 100%;
     background-color: bg;
     color: primary-text;
+    font-family: default;
```

Comment:
Nop

Reply:
This line alone load the Inter as the default font. If I remove it, this PR is useless